lib/linguist/languages.yml

```diff
@@ -1244,6 +1244,16 @@ Emacs Lisp:
   codemirror_mode: commonlisp
   codemirror_mime_type: text/x-common-lisp
   language_id: 102
+Email:
+  type: data
+  extensions:
+  - ".eml"
+  - ".email"
```

Comment:
[Looks like a lot of the `.email` files are not EML](https://github.com/search?p=5&q=NOT+reply-to+extension%3Aemail&type=Code). There's few benefit to add support for data languages anyway, so I think it'd be better to remove that file extension from the pull request.